Repositories/BookingRepositoty.cs

```diff
@@ -0,0 +1,73 @@
+﻿using AirportTicketBooking.Classes;
+
+namespace AirportTicketBooking.Controllers
+{
+    internal class BookingRepositoty
+    {
+        private const string filePath = "booking.csv";
+
+
+        internal static void SaveToFile(Booking booking)
```

Comment:
I prefer making the name more generic, for example: `Save` only. This would be helpful when changing the implementation or changing the storage type (upgrading to db instead of file storage). 

Also, please consider using interfaces for the repositories to have a generic contract for your work.

The same thing should be applied to all other repositories.